src/smartactions/ScheduledTransfers.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import { IERC7579Execution } from "modulekit/Accounts.sol";
+import { SchedulingBase } from "./SchedulingBase.sol";
+
+abstract contract ScheduledTransfers is SchedulingBase {
```

Comment:
I think we should have a scheduled token transfer too right?

Reply:
Yep, we have `ScheduledOrders` contract for that